base/install_motor.sh

```diff
@@ -34,3 +33,45 @@ cd ..
 
 make -j${JOBS}
 make clean
+
+#Add pmac module
+
+cd ..
+
+git clone --depth 1 --branch ${PMAC_VERSION} \
+    https://github.com/dls-controls/pmac
+
+cd pmac
+
+rm -rf .git/ .git*
+
+module_releases="PMAC=${EPICS_MODULES_PATH}/pmac
+"
+echo "$module_releases" >> ${EPICS_MODULES_PATH}/../RELEASE
+
+echo "
+" > configure/RELEASE.local.linux-x86_64
+
+echo "
+EPICS_BASE=/opt/epics/base
+ASYN=${EPICS_MODULES_PATH}/asyn
+CALC=${EPICS_MODULES_PATH}/calc
+MOTOR=${EPICS_MODULES_PATH}/motor
+BUSY=${EPICS_MODULES_PATH}/busy
+" > configure/RELEASE.local
+
+echo "
+BUILD_IOCS=NO
+USE_GRAPHICSMAGICK=NO
+
+SSH             = $(pkg-config --variable=prefix libssh2)
+SSH_LIB         = $(pkg-config --variable=libdir libssh2)
+SSH_INCLUDE     = -I$(pkg-config --variable=includedir libssh2)
+
+WITH_BOOST = NO
+
+" > configure/CONFIG_SITE.linux-x86_64.Common
```

Comment:
I'd rather remove `CONFIG_SITE.linux-x86_64.Common` and output to `CONFIG_SITE.local` instead. Same rationale from the comment above.

Reply:
The `CONFIG_SITE` file does not include `CONFIG_SITE.local`. To do this we would have to remove `CONFIG_SITE.linux-x86_64.Common`, edit `CONFIG_SITE` and create `CONFIG_SITE.local`. Would we be making things more confusing?

Reply:
I like the implementation you did writing directly to `CONFIG_SITE` instead.